### DISCUSSION TITLE
IS IMPLEMENTED: module suggestion: search item on account (inventory etc.)

### DISCUSSION THREAD

**Discussion by Taschenbuch** [blish-hud/Blish-HUD#639] [Module Ideas]
Has been suggested in the discord already. 

like gw2timer or gw2effiency account search but as a module so that you have it as a small compact tool ingame instead of a huge website. picture for reference
![grafik](https://user-images.githubusercontent.com/43114787/158219750-65974a99-972a-4607-92fc-fd60f8626a5b.png)

how it should work:
type in an item name or a part of an item name. Click a search button to see where this item is on the account. 

where it should look:
- trading post SELL orders 
- legendary armory
- material storage
- bank
- shared inventory slots
- inventories of all characters of the account
- active equip templates of all characters of the account
- inactive equip templates of all characters of the account
- jade bot equip panel of all characters of the account
- fishing equip panel of all characters of the account

Notes:
- Consider, that it can be in multiple places on the account in different amounts and seperated in multiple stacks. e.g. inventory has a stack of 5 iron ore and next to it a stack of 20 iron ore and the bank contains 10 iron ore. In this case the search result should list that there are 25 iron ore in the inventory of character X and 10 iron ore in the bank.

gold plating
- show item icon: maybe that is overkill because all of that has to be downloaded and would bloat up this module. It should be fast and not slow down just because it wants to look fancy
- filter: not only search for item name but search for certain item types (armor) and so on. check gw2efficency account search for inspirations

**Comment by tentose** [comment]
I'm going to give this a try.

**Comment by tentose** [comment]
Check out: https://github.com/tentose/Gw2ItemSearch/releases/tag/0.2.0-4-main
